lms/static/sass/course/ccx_coach/_dashboard.scss

```diff
@@ -44,12 +49,99 @@ table.ccx-schedule {
     border: 1px solid #cbcbcb;
     padding: 15px;
     margin-bottom: 20px;
+    button {
+        display: inline-block;
+        vertical-align: middle;
+        margin-right: .3125rem;
+        margin-bottom: .625rem;
```

Comment:
I took this code from https://github.com/edx/ux-pattern-library
they are using rem